tests/e2e/vm_creation_performance_test.go

```diff
@@ -0,0 +1,228 @@
+package e2e
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/deckhouse/virtualization/api/client/kubeclient"
+	"github.com/deckhouse/virtualization/api/core/v1alpha2"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/spf13/pflag"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/ptr"
+	"log"
+	"time"
+)
+
+func CVMI(client kubeclient.Client, name string, action string) (*v1alpha2.ClusterVirtualMachineImage, error) {
+
+	cvmi := v1alpha2.ClusterVirtualMachineImage{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: name,
+		},
+		Spec: v1alpha2.ClusterVirtualMachineImageSpec{
+			DataSource: v1alpha2.CVMIDataSource{Type: "HTTP", HTTP: &v1alpha2.DataSourceHTTP{
+				URL: "https://cloud-images.ubuntu.com/jammy/current/jammy-server-cloudimg-amd64.img"},
+			},
+		},
+	}
+
+	if action == "create" {
+		res, err := client.ClusterVirtualMachineImages().Get(context.TODO(), name, metav1.GetOptions{})
+		if res != nil && err == nil {
+			return res, err
+		}
+
+		res, err = client.ClusterVirtualMachineImages().Create(context.TODO(), &cvmi, metav1.CreateOptions{})
+		if err != nil {
+			log.Fatal(err)
+		}
+		return res, err
+	}
+
+	if action == "delete" {
+		res, err := client.ClusterVirtualMachineImages().Get(context.TODO(), name, metav1.GetOptions{})
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		err = client.ClusterVirtualMachineImages().Delete(context.TODO(), res.Name, metav1.DeleteOptions{})
+		if err != nil {
+			log.Fatal(err)
+		}
+		fmt.Println(res.Name, "was deleted")
```

Comment:
I recommend to use GinkgoWriter, so these messages can be enabled with `-v` flag or disabled if not needed.
See https://onsi.github.io/ginkgo/#logging-output